README.md

```diff
@@ -19,7 +20,32 @@ pip install -e ".[dev]"
 
 # User API, subject to change
 
-## single GPU
+We provide two types of scaling for per-tensor scaling of tensors: dynamic and delayed.
```

Comment:
This sounds kinda of weird, what "We provide two scaling strategies: per-tensor dynamic and delayed."